rfcs/029-resource-notices.md

```diff
@@ -0,0 +1,101 @@
+- Feature Name: Resource Notices
+- Start Date: 2021-03-22
+- RFC PR: [amundsen-io/rfcs#29](https://github.com/amundsen-io/rfcs/pull/29) (after opening the RFC PR, update this with a link to it and update the file name)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# Resource Notices
+
+## Summary
+
+Resource Notices allows Amundsen administrators to show informational, warning, and alert messages related to the different resources (tables, dashboards, people) we expose in Amundsen.
+
+## Motivation
+
+This feature will help administrators show messages related to deprecation, updates (or lack of), and informational messages related to specific resources. In Lyft's case, we use them to point final users to improved versions of current surfaced tables, so final users employ more trusted data.
+
+## Guide-level Explanation (aka Product Details)
+
+An Amundsen final user can see a notice in the table/dashboard/user detail page whenever their system admin adds one related to it.
```

Comment:
Will there be an API to create notices? For example, a side-effect of a SEV is the automatic creation of a notice for the impacted resources.

Reply:
It wasn't anything we were thinking at the beginning, but it has been a common suggestion so we added it on the phase 3 of the roadmap below.